Add osu! Beatmapping World Championship 2021 Judge Applications newspost and wiki article (#5235)
ppy/osu-wiki: news/2021-04-11-osu-beatmapping-world-championship-2021-call-for-judges.md

---
layout: post
title: "osu! Beatmapping World Championship 2021: Call for Judges"
date: 2021-04-11 14:00:00 +0000
---

> With the Spring season comes the next iteration of osu! Beatmapping World Championship! If you're excited and eager to help us out, here's your chance: We are looking for judges!

Reviewer: ```suggestion
With the Spring season comes the next iteration of osu! Beatmapping World Championship! If you're excited and eager to help us out, here's your chance, because we're looking for judges!
```
nit

![](/wiki/shared/news/banners/obwc2021.png)

> For the third iteration of osu! Beatmapping World Championship, we have taken your feedbacks into account and implemented some significant changes in order to make this contest enjoyable for everyone. To help us with judging, we are looking for mappers, modders, and this time even players! Looking for where to apply? Just keep on reading, and you'll find out!

Reviewer: ```suggestion
For the third iteration of osu! Beatmapping World Championship, we have taken your feedback into account and implemented some significant changes in order to make this contest enjoyable for everyone. To help us with judging, we are looking for mappers, modders, and this time even players! Looking for where to apply? Just keep on reading, and you'll find out!
```

## osu! Beatmapping World Championship 2021

> For more information, check out the [wiki page](/wiki/Contests/oBWC/3) and the [official website](https://obwc.net/).

Reviewer: ```suggestion
For more information, check out the [wiki page](/wiki/Contests/oBWC/3) and the [official website](https://obwc.net/).
```
official website needs alt text; i can't seem to access the page for some reason tho (connection closed)

### Contest Judges - Mappers

Unlike the last iteration, this year's contest is planned to host an accumulative leaderboard, enabling all participants to stay in the contest for as long as it goes. To determine the placements, we will need the help of the mappers extraordinaire, modding enthusiasts, and maybe even you (yes, you!) to partake as judges throughout the contest.

> Now, you may have questions as to according to which criteria these judges will rate the submissions. Worry not! Here are the criteria:

Reviewer: ```suggestion
Now, you may have questions reagrding which criteria these judges will rate the submissions. Worry not! Here are the criteria:
```
as to accord to which?

Reviewer 2: "regarding"

> - **Fundamentals**: This criterion will focus on the fundamental abilities of mapping. A map can show the fundamental skills of mappers in different aspects, include, but not limit to: respect to the ranking criteria, completeness of hitsounding, style and pattern consistencies throughout the difficulty, reasonable fluctuations of difficulty according to the music, and many more.

Reviewer: ```suggestion
- **Fundamentals**: This criterion will focus on the fundamental abilities of mapping. A map can show the fundamental skills of mappers in different aspects, including, but not limited to: respect to the ranking criteria, completeness of hitsounding, style and pattern consistencies throughout the difficulty, reasonable fluctuations of difficulty according to the music, and much more.
```

- **Uniqueness**: Throughout the history of osu!, we can see that beatmaps displaying unique patterns usually be the centre of community attention and sometimes even become era-defining masterpieces. With that in mind, we would like to draw players' attention to the artistic aspects of mapping and see where their opinions revolve.

Additionally, we will also entertain a **Player's Impression** section. Players are asked to note down their personal thoughts and subjective opinions regarding the beatmap and rate it accordingly.

To take part as a Players' Pick judge, your osu! rank needs to be *higher than #5000* as the initial requirement. We're planning to recruit **5 judges** in total for this section of the contest. So if you're a professional osu! player and like playing a variety of beatmaps, don't miss out on becoming a Players' Pick judge!

### Multi-mode Judges

> Drummers, fruit catchers, and keyboard smashers behold! This year, we are going to host three more contests for three other game modes. The details are still worked on, but if you're an early bird wishing to partake as a judge for osu!taiko, osu!catch, or osu!mania edition of the osu! Beatmapping World Championship, you may submit your application right away!

Reviewer: ```suggestion
Drummers, fruit catchers, and keyboard smashers behold! This year, we are going to host three more contests for three other game modes. The details are still worked on, but if you're an early bird wishing to partake as a judge for the osu!taiko, osu!catch, or osu!mania edition of the osu! Beatmapping World Championship, you may submit your application right away!
```

## Schedule

> Below is the provisional schedule for the upcoming contest. Be reminded that reschedules might occur due to unexpected organizational matters.

Reviewer: ```suggestion
Below is the provisional schedule for the upcoming contest. Keep in-mind that reschedules might occur due to unexpected organizational matters.
```
nit

Reviewer 2: "keep in mind"? why hyphenate?

Reviewer: afaik "in-mind" and "in mind" are both correct; just down to preference of style (like video game vs videogame vs video-game); though could be totally wrong

| Event | Timestamp |
| --: | :-- |
| o!BWC Announcement | 2021-05-02 |
| Registration Phase | 2021-05-02/2021-05-16 |
| First Round - Mapping phase | 2021-05-17/2021-05-31 |
| First Round - Judging phase | 2021-06-01/2021-06-15 |
| Second Round - Mapping phase | 2021-06-16/2021-06-30 |
| Second Round - Judging phase | 2021-07-01/2021-07-15 |
| Third Round - Mapping phase | 2021-07-16/2021-07-30 |
| Third Round - Judging phase | 2021-07-31/2021-08-14 |
| Fourth Round - Mapping phase | 2021-08-15/2021-08-29 |
| Fourth Round - Judging phase | 2021-08-30/2021-09-13 |
| Results Announcement & Livestream | 2021-09-15 |

Multi-mode contests are planned to be hosted approximately 2 months anterior to the conclusion of the osu! Beatmapping World Championship. We are trying our best to finalize things in that regard as quick as possible, so keep a lookout!

## Applications

Interested in becoming a judge? Awesome! Whichever judging section you wish to take part in, all you have to do is to click on the link below.

> [**Please fill out this form in order to apply as a judge!**](https://forms.gle/he4sT7DeX5UVr4j27)

Reviewer: ```suggestion
[**Please fill out this form in order to apply as a judge!**](https://forms.gle/he4sT7DeX5UVr4j27 "Google Forms")
```

---

> Ultimately, it is time to wrap things up. We hope that 2021 edition of osu! Beatmapping Championship, with its traditional and brand-new features, has piqued your interest. For forthcoming updates, [follow us on Twitter](https://twitter.com/osubwc) and [join our Discord server](https://discord.gg/CZp4bNx)!

Reviewer: ```suggestion
Unfortunately, it is time to wrap things up. We hope that the 2021 edition of osu! Beatmapping Championship, with its traditional and brand-new features, has piqued your interest. For forthcoming updates, [follow us on Twitter](https://twitter.com/osubwc "Twitter") and [join our Discord server](https://discord.gg/CZp4bNx "Discord")!
```
assuming "ultimately" meant "in conclusion", but either way sounds a little off?

Reviewer 3: Maybe say, "With all the exciting news, it is time to wrap up."?
